src/health/health.controller.ts

```diff
@@ -1,12 +1,19 @@
 import { Controller, Get } from '@nestjs/common';
 import { HealthService } from './health.service';
 import { StatusInterface } from './interfaces/status.interface';
+import { ApiResponse } from '@nestjs/swagger';
+import { Health } from './health';
 
 @Controller('health')
 export class HealthController {
   constructor(private healthService: HealthService) {}
 
   @Get()
+  @ApiResponse({
+    status: 201,
```

Comment:
Why use 201 and not 200 for health check ?

Reply:
The right code is 200, was a typing mistake here